src/index.js

```diff
@@ -52,7 +53,6 @@ class Server {
         this.jwsValidator = {};
     }
 
-
```

Comment:
@vijayg10 please dont just remove whitespace because it doesnt fit your formatting preference. So long as the file passes the linting rules you should not mess with formatting.